data/api/client-server/definitions/key_backup_session_data.yaml

```diff
@@ -0,0 +1,62 @@
+# Copyright 2022 The Matrix.org Foundation C.I.C
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+type: object
+title: SessionData
+description: |-
+  The format of a backed-up session key, prior to encryption, when using the
+  `m.megolm_backup.v1.curve25519-aes-sha2` algorithm.
+properties:
+  algorithm:
+    type: string
+    description: |-
+      The end-to-end message encryption algorithm that the key is for. Must be `m.megolm.v1.aes-sha2`.
```

Comment:
Descriptions in this file have been inspired by the changes in https://github.com/matrix-org/matrix-spec/pull/1088